ru/turing/courses/lesson2/animals/bird.java

```diff
@@ -0,0 +1,18 @@
+package ru.turing.courses.lesson2.animals;
+
+public class bird extends Animal {
```

Comment:
```suggestion
public class Bird extends Animal {
```
Называем классы с большой буквы, переменные и методы - с маленькой